newrelic_lambda_cli/layers.py

```diff
@@ -178,9 +191,10 @@ def install(
         nr_license_key,
         allow_upgrade,
         enable_extension,
+        enable_function_logs,
     )
 
-    if not update_kwargs:
+    if not update_kwargs or type(update_kwargs) != dict:
```

Comment:
What is the scenario being checked here?

Reply:
Also, could be `isinstance(update_kwargs, dict)`.

Reply:
Below that line, we use `update_kwargs` as kwargs. Often, it's a boolean value, rather than actually being kwargs. It can be either False (which worked before), or it can be True, which causes a runtime exception due to type incompatibility. This check avoids that runtime exception.